ajax/libs/clone/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "clone",
+  "filename": "clone.min.js",
+  "description": "deep cloning of objects and arrays",
+  "keywords": [
+    "clone",
+    "object",
+    "array",
+    "function",
+    "date"
+  ],
+  "version": "2.1.1",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/pvorb/clone"
+  },
+  "author": "Paul Vorbach <paul@vorba.ch> (http://paul.vorba.ch/)",
+  "license": "MIT",
+  "autoupdate": {
```

Comment:
@pvnr0082t That makes sense since I consider a release published once it's on npm, not GitHub.